tsdb/compact.go

```diff
@@ -755,11 +755,11 @@ func (c DefaultBlockPopulator) PopulateBlock(ctx context.Context, metrics *Compa
 
 	for symbols.Next() {
 		if err := indexw.AddSymbol(symbols.At()); err != nil {
-			return errors.Wrap(err, "add symbol")
+			return fmt.Errorf("add symbol: %w", err)
 		}
 	}
 	if symbols.Err() != nil {
```

Comment:
nit: we could store the error here

Reply:
There are several places like that, is there a linter for that ?

Reply:
I don't know

Reply:
```suggestion
	if err := symbols.Err(); err != nil {
```